projects/angular-split/src/lib/interface.ts

```diff
@@ -5,13 +5,15 @@ export interface IPoint {
   y: number
 }
 
+export type IAreaSize = number | '*'
```

Comment:
weird to prefix a type (not an interface) with the capital `I` prefix that means Interface :P

Reply:
Good point @lf-novelt - I'm not a fan of the I for interfaces either, it's more a matter of keeping consistency.